drivers/UARTSerial.h

```diff
@@ -234,6 +234,18 @@ class UARTSerial : private SerialBase, public FileHandle, private NonCopyable<UA
      */
     void set_format(int bits = 8, Parity parity = UARTSerial::None, int stop_bits = 1);
 
+    /** get the available characters in the RX ringbuffer*/
+    uint32_t get_rx_buffer_size()
```

Comment:
shouldn't this be const methods? and possibly `size_t` or are the sizes in uint_32t?

Reply:
size_t is a typedef for uint32_t, and yes, I will make them consts